src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -848,6 +887,29 @@ def do_request(params)
         it { expect(assigns(:errors)).not_to be_nil }
         it { expect(response).to have_http_status(:ok) }
       end
+
+      context 'for multibuild package' do
+        let(:params) {
+          { project:    source_project,
+            package:    "#{source_package}:multibuild-package",
+            repository: repo_leap_42_2.name,
+            arch:       architecture.name}
+        }
+
+        context 'instance variables' do
```

Comment:
This is not really what a "context" is, "describe" would be a more appropriate word here but I would just get rid of it all together.